docs/hosting/app.md

```diff
@@ -124,28 +117,31 @@ reflex deploy --project f88b1574-f101-####-####-5f########## --region sjc --regi
 By default all apps are deloyed in `sjc` if no other regions are given. If you wish to deploy in another region or several regions you can pass the `--region` flag (`-r` also works) with the region code. Check out all the regions that we can deploy to below:
 
 
-```python eval
-rx.table.root(
-    rx.table.header(
-        rx.table.row(
-            rx.table.column_header_cell("Region Code"),
-            rx.table.column_header_cell("Region"),
-        ),
-    ),
-    rx.table.body(
-        *[
-            rx.table.row(
-                rx.table.cell(rx.code(region_code, style=get_code_style("violet"))),
-                rx.table.cell(region_name, style=cell_style),
-            )
-            for region_code, region_name in regions.items()
-        ]
-    ),
-    variant="surface",
-)
+## Config File
+
+To create a `config.yml` file for your app run the command below:
+
+```bash
+reflex cloud config
+```
+
+This will create a yaml file similar to the one below where you can edit the app configuration:
+
+```yaml
+name: default
+description: this is a default app
+regions:
+- sjc
```

Comment:
```suggestion
regions:
  sjc : 1
```

This should be a dict than a list, where `sjc` is the region and `1` is the number of machines to scale